packages/salesforcedx-sobjects-faux-generator/test/fauxgeneratortest.ts

```diff
@@ -0,0 +1,135 @@
+import * as chai from 'chai';
+import * as fs from 'fs';
+import { FauxClassGenerator } from '../src/generator/fauxClassGenerator';
+
+const expect = chai.expect;
+
+describe('generate fields set', function() {
```

Comment:
Do you clean up the generated folders?

Reply:
no, good catch - when we regenerate, I need to blow away the old files since I don't want to try to track exactly what has changed, etc....    that's a bug

Reply:
fixed